test/TestUtils.py

```diff
@@ -185,3 +186,32 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+def test_get_yaml_files_no_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.setenv('PATH', '')
+    utils.get_yaml_files(options)
+    stdout = capsys.readouterr().out.split('\n')
+    assert stdout[0] == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert stdout[1].startswith('Warning: Failed to locate command:')
+
+
+def test_get_yaml_files_not_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.chdir('/tmp')
+    utils.get_yaml_files(options)
+    stdout = capsys.readouterr().out.split('\n')
+    assert stdout[0] == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert stdout[1].startswith('Warning: Failed to discover yaml files to lint using git:')
```

Comment:
I think this is missing the check about the amount of files.

Reply:
Also, we probably need another test with rules but no git repo.
You can use the [`tmp_path` fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture) for getting a tmp dir, copy rules there and `monkeypatch.chdir()` to that location.